### PR TITLE
Allow error reports to be sent from monitoring to any Slack channel via Cypress Dashboard

### PR DESCRIPTION
**Pull request by wdsrocha**
This PR has the objective to alert when some monitoring test fails on our selected Slack channels, instead of sending to the default Slack channel where every other test goes.

One way to do this is to send the tests to Cypress Dashboard, which automatically sends any failures to the Slack channel. Given that the monitoring checkout-ui-tests container already has the Cypress Dashboard [project id](https://github.com/vtex/checkout-ui-tests/blob/master/src/monitoring/index.js#L89) and [access key](https://github.com/vtex/checkout-ui-tests/blob/master/cypress.json#L10) (or is the access key [here](https://github.com/vtex/checkout-ui-tests/blob/9034681f36d6500774e0ba414f4ddcb6d0aca1fb/package.json#L55-L57)?), all I think needs to be done is to pass the `key` and `recording` parameters to the `index.js` file configuration. Also, we need to use the current environment as a tag, as Cy Dashboard redirects the test to slack based on tags.

Btw, yes, the key shouldn't be explicit and I think we should fix this soon.

### Downsides

On monitoring, each test is executed in a unique run, so this will make the dashboard have _lots_ of runs. As an example, see the picture below.

image

## How to manually test this

I'm not sure, as we still don't know how the tests are executed in VTEX infrastructure. I've heard that all that infra does is put a cron on a docker container, so I suppose that running `yarn test:ci-stable` will simulate the same thing that happens in the infra, as this command run the docker container as well.

In short, just clone this repository and run `yarn test:ci-stable` (or `yarn test:stable` if you want to test faster without the container). Open the https://dashboard.cypress.io/projects/kobqo4/runs and check for the results.

For example, I executed this test on [this run](https://dashboard.cypress.io/projects/kobqo4/runs/2158/overview) and it sent the error [here](https://vtex.slack.com/archives/CM4KFG8RF/p1622475784053800).

### CODE REVIEW

File: src/monitoring/index.js [hunk 1]
```diff
@@ -87,6 +87,9 @@ const CYPRESS_CONFIG = {
     video: !program.skipUpload,
   },
   projectId: 'kobqo4',
+  key: '6ecaeca9-8292-49d5-b954-f5d330526703',
```
Comment (gustavo-almeida) [inline src/monitoring/index.js:90]: É seguro expor essa chave?

Comment (wdsrocha) [reply src/monitoring/index.js:90]: Não é 😕, depois temos que passar a utilizar variáveis de ambiente e gerar outra chave.

O único motivo por qual decidi expor a chave _atualmente_ é porque ela já foi exposta em outros dois lugares: [1](https://github.com/vtex/checkout-ui-tests/blob/master/cypress.json#L10), [2](https://github.com/vtex/checkout-ui-tests/blob/9034681f36d6500774e0ba414f4ddcb6d0aca1fb/package.json#L55-L57)

**Review by silviasfon — APPROVED** [review]

**Review by lucasecdb — APPROVED** [review]

**Review by gustavo-almeida — APPROVED** [review]
LGTM
